core/domain/exp_jobs_one_off.py

```diff
@@ -393,7 +393,8 @@ def map(item):
 
             change_list = commit.commit_cmds
             # Check if commit is to revert the exploration.
-            if change_list[0]['cmd'].endswith('revert_version_number'):
+            if change_list and change_list[0]['cmd'].endswith(
```

Comment:
I'll merge this, but just a quick question. If change_list is empty, then this will fall into the else branch. Will that cause a failure?